### DISCUSSION TITLE
0x9adfacdc96bd490df151ab22162ede3b28a259009233445cef06c53b160728ae   proposal id for Tangible Non Fungible KCC Node. tnf-node-1

### DISCUSSION THREAD

**Discussion by ZinCoin** [kcc-community/any-advice-issue#36] [Q&A]
Tangible Non Fungible Team is excited to help support the kcc network. We have submitted out proposal @ block 10528312 and are ready to stake!

While we have surpassed the minimum for staking on our node, we will continue to acquire kcs while we wait for the proposal to be voted on so our active node can be verified so we can stake.

**Comment by Bandar511r** [comment]
I'm waiting for you

**Reply by srbaeza-ai** [reply]
- [ yes
]

**Comment by srbaeza-ai** [comment]
How to start
